pkg/cloud/ibmcloud/actuators/machine/actuator.go

```diff
@@ -161,7 +162,8 @@ func (ic *IbmCloudClient) Delete(ctx context.Context, cluster *clusterv1.Cluster
 // Update updates a machine and is invoked by the Machine Controller
 func (ic *IbmCloudClient) Update(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	log.Printf("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
```

Comment:
klog here?

Reply:
I did not change all parts from `log` to `klog`. So, I just change on demand for this ticket. 
A ticket for change `log` to `klog` was logged for this issue.